package.json

```diff
@@ -62,10 +63,13 @@
     "renderer-docker": "(cd ../renderer-docker && ./start_docker.sh)",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "prestart": "npm run modernizr",
+    "prebuild": "npm run modernizr",
     "build": "react-scripts build",
     "rebuild_renderer": "(cd ../renderer-docker && ./rebuild_docker.sh)",
     "lint": "eslint src/ --ext ts,tsx",
-    "postbuild": "node postbuild.js"
+    "postbuild": "node postbuild.js",
+    "modernizr": "npx modernizr -c modernizr-config.json -d public/modernizr.min.js"
```

Comment:
Still not a fan of npx, if someone removed  from package.json not realizing it is used then instead of failing it would just use a different version

Reply:
Nothing to do with being a fan of npx, it just has no purpose being inside the package.json, I just copied it from the commands I was running outside it.